MSGraphCoreSDK/MSGraphCoreSDK/HTTPClient/MSMiddlewareFactory.h

```diff
@@ -14,7 +14,8 @@ typedef NS_ENUM(NSInteger, MSMiddlewareType)
 {
     MSMiddlewareTypeAuthentication,
     MSMiddlewareTypeRedirect,
-    MSMiddlewareTypeHTTP
+    MSMiddlewareTypeHTTP,
+    MSMiddlewareTypeRetry
```

Comment:
please check if values of enum is important, else keep these in alphabetical order.